docs/release-notes/2.31.0.md

```diff
@@ -0,0 +1,159 @@
+# Percona Monitoring and Management 2.31.0
```

Comment:
@rasika-chivate please also add changes for the left menu, RNs aren't visible there

Reply:
Yes. I will make the changes.